packages/client/src/components/Button/button.scss

```
@@ -57,3 +57,15 @@
 		background: var(--colorGray);
 	}
 }
+
+.buttonPoginations {
```

Думаю, лучше так написать:
```suggestion
.buttonPage {
```